raiden-cli/src/raiden.ts

```diff
@@ -63,7 +63,7 @@ export default class RaidenService {
 
   get allChannels(): RaidenChannel[] {
     // To resolve structure {token: {partner: [channel..], partner:...}, token...}
-    return _.flatMap(_.values(this._channels), (partnerChannels) => _.values(partnerChannels));
+    return flatMap(values(this._channels), (partnerChannels) => values(partnerChannels));
```

Comment:
`flatMap` here can also be replaced by `Array.prototype.reduce` to avoid the import as well... something like:
```ts
Object.entries(this._channels).reduce((acc, partnerChannels) => acc.concat(Object.values(partnerChannels)), [] as RaidenChannel[]);
```
Lodash is very useful sometimes, but there's a lot of things we don't need it for on ES6 or even old JS.

Reply:
Replace `entries` with `values` and it works. :grimacing:

Reply:
Replace `entries` with `values` and it should work. :D

Reply:
:heavy_check_mark: